content/source/docs/enterprise/api/state-version-outputs.html.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: enterprise2
+page_title: "State Version Outputs - API Docs - Terraform Enterprise"
+sidebar_current: "docs-enterprise2-api-state-version-outputs"
+---
+
+[200]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200
+[404]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/404
+[JSON API document]: /docs/enterprise/api/index.html#json-api-documents
+[JSON API error object]: http://jsonapi.org/format/#error-objects
+
+# State Version Outputs API
+
```

Comment:
Can you include a brief description as to what a state version output is?

Reply:
Added!